analysis/bulk-deconvolution/scripts/run-quantiseq.R

```diff
@@ -0,0 +1,81 @@
+# This script runs quanTIseq for samples in a given ScPCA project and exports 
+#  a TSV of cell type proportions for each sample in the project.
+
+renv::load()
+library(optparse)
+library(quantiseqr)
+
+# Parse options --------
+option_list <- list(
+  make_option(
+    "--project_id",
+    type = "character",
+    default = "SCPCP000001",
+    help = "ScPCA project id to run quanTIseq on."
+  ),
+  make_option(
+    "--input_dir",
+    type = "character",
+    default = here::here("analysis", "bulk-deconvolution", "data", "tpm"),
+    help = "Input directory containing a project-specific TPM matrices saved as rds files named <project id>-tpm.rds."
+  ),
+  make_option(
+    "--output_dir",
+    type = "character",
+    help = "Output directory to quanTIseq inference TSV."
+  )
```

Comment:
Just as a preference, I tend to like these kinds of scripts to just take an input and output file name, rather than trying to rigorously control the file name patterns. I find it tends to make things more modular and flexible, and also keeps things a bit more transparent in the wrapper shell script.

Reply:
I can generally update this in the tpm script too.

Reply:
Followup: would you prefer then to not pass in `project_id` altogether but _only_ the tpm file name, for example?

Reply:
If you don't use the project id in the script, then no need, right?

Reply:
Would you consider it an _overly_ recreational revision to have a separate PR that generally updates other scripts for these sorts of conventions? I do agree explicit input/output files all around are more robust.

Reply:
The only other one I would change for now is `calculate-tmp.R`.